src/lib/Config.ts

```diff
@@ -0,0 +1,120 @@
+export default class Config {
+
+    // Signal if the app should be started in offline mode
+    public static offline = navigator.onLine !== undefined && !navigator.onLine;
+
+    // When packaged as distributed offline app, subdomains are folder names instead
+    public static offlinePackaged = false;
+
+    /* Public methods */
+
+    public static origin(subdomain: string) {
+        return Config._origin(subdomain);
+    }
+
+    public static src(subdomain: string) {
+        return Config._origin(subdomain, true);
+    }
+
+    public static get tld() {
+        const tld = Config._myOrigin.split('.');
+        return [tld[tld.length - 2], tld[tld.length - 1]].join('.');
+    }
+
+    public static get network() {
+        if (Config.offlinePackaged) { return 'main'; }
+
+        switch (Config.tld) {
+            case 'nimiq.com': return 'main';
+            case 'nimiq-testnet.com': return 'test';
+            default: return 'test'; // Set this to 'test', 'bounty', or 'dev' for localhost development
+        }
+    }
+
+    public static get cdn() {
+        if (Config.offlinePackaged) { return Config.src('keyguard') + '/nimiq.js'; }
```

Comment:
I don't understand the rationale for `keyguard` here?
Perhaps it is just because of the current offline structure?

Reply:
Yes, this was from how the offline package was packaged for the current Safe, where the Nimiq lib was included in the Keyguard folder.